ReactNative/ReactTwitter/core/views/main-navigator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {
+  Navigator
+} from 'react-native'
+
+var DebugScreenView = require('./debug-screen.js')
+var DeepLinkingView = require('./deep-linking.js')
+
+const debugScreenID = 'debug-screen-identifier'
```

Comment:
Ideally we would like to define these constants inside our ViewComponents. Any better solutions to avoid these identifier hardcoded here?